LogBundle/Repository/LogRepositoryInterface.php

```diff
@@ -2,35 +2,13 @@
 
 namespace OpenOrchestra\LogBundle\Repository;
 
+use OpenOrchestra\ModelInterface\Repository\PaginateRepositoryInterface;
+
 
 /**
  * Class LogRepositoryInterface
  */
-interface LogRepositoryInterface
+interface LogRepositoryInterface extends PaginateRepositoryInterface
```

Comment:
Is it coherent to keep an empty interface ? maybe you could remove it and replace it where it was used by PaginateRepositoryInterface.
Or you could mark it as deprecated to remove it on the next tag